### ISSUE TITLE
Problems after Slackware -current upgrade 22 Nov

### ISSUE THREAD

**Issue by hamkg** [mateslackbuilds/msb#26]
### Slackware Version 
-current

### MATE Version 
1.20.2

### Description 
After today's slackware -current upgrade, my MATE desktop is slightly lagging. Also, the clock has disappeared from the panel and can't be reinserted. Could it be that the upgrade also upgraded vala from 0.40 (part of MATE) to 0.42 (part of Slackware -current? Or some other upgrade?

**Comment by willysr** [comment]
yes, you need to remove vala from MSB and install vala which is part of the Slackware-Current

**Comment by hamkg** [comment]
I have upgraded Vala to 0.42 - it is after that upgrade and reboot that the clock has gone, and the desktop and panel loads slower than before after logging in.

**Comment by willysr** [comment]
can you list the mate-* packages in your system?
my guess is that you haven't upgrade to the latest one which may still linked to the previous library version.

Just wondering, did you update your machine periodically according to the -current changelog or this is the first update after some time?

**Comment by hamkg** [comment]
Here's the list.
I normally upgrade at least once a week, sometimes more often (except kernel and glibc stuff).
Maybe I should try to rebuild the mate-applets package?

``bash-4.4$ ls -l /var/log/packages/mate*
-rw-r--r-- 1 root root  93120 Jun  1 13:16 /var/log/packages/mate-applets-1.20.2-x86_64-1_msb
-rw-r--r-- 1 root root  13151 Feb  7  2018 /var/log/packages/mate-backgrounds-1.20.0-noarch-1_msb
-rw-r--r-- 1 root root  12314 Jun 25 12:12 /var/log/packages/mate-calc-1.20.2-x86_64-1_msb
-rw-r--r-- 1 root root   1419 Feb  7  2018 /var/log/packages/mate-common-1.20.0-x86_64-1_msb
-rw-r--r-- 1 root root  46178 Jun 25 12:12 /var/log/packages/mate-control-center-1.20.3-x86_64-1_msb
-rw-r--r-- 1 root root  18441 Jun  5 10:13 /var/log/packages/mate-desktop-1.20.3-x86_64-1_msb
-rw-r--r-- 1 root root 219661 Oct 22 08:43 /var/log/packages/mate-icon-theme-1.20.2-noarch-1_msb
-rw-r--r-- 1 root root 463091 Feb  7  2018 /var/log/packages/mate-icon-theme-faenza-1.20.0-noarch-1_msb
-rw-r--r-- 1 root root  18737 Jun 25 12:12 /var/log/packages/mate-media-1.20.1-x86_64-1_msb
-rw-r--r-- 1 root root  15658 Jun 25 12:12 /var/log/packages/mate-menus-1.20.1-x86_64-1_msb
-rw-r--r-- 1 root root  15323 Jun 25 12:12 /var/log/packages/mate-notification-daemon-1.20.1-x86_64-1_msb
-rw-r--r-- 1 root root  31187 Jun  1 13:17 /var/log/packages/mate-panel-1.20.2-x86_64-1_msb
-rw-r--r-- 1 root root  12187 Jun 25 12:12 /var/log/packages/mate-polkit-1.20.1-x86_64-1_msb
-rw-r--r-- 1 root root  53995 Jun 25 12:13 /var/log/packages/mate-power-manager-1.20.2-x86_64-1_msb
-rw-r--r-- 1 root root  14638 Oct 22 08:43 /var/log/packages/mate-screensaver-1.20.2-x86_64-1_msb
-rw-r--r-- 1 root root  19830 Jun 25 12:13 /var/log/packages/mate-sensors-applet-1.20.2-x86_64-1_msb
-rw-r--r-- 1 root root  17836 Jun 25 12:13 /var/log/packages/mate-session-manager-1.20.1-x86_64-1_msb
-rw-r--r-- 1 root root  22213 Jun  5 10:13 /var/log/packages/mate-settings-daemon-1.20.3-x86_64-1_msb
-rw-r--r-- 1 root root  18433 Apr 25  2018 /var/log/packages/mate-system-monitor-1.20.0-x86_64-2_msb
-rw-r--r-- 1 root root  19175 Jun 25 12:13 /var/log/packages/mate-terminal-1.20.1-x86_64-1_msb
-rw-r--r-- 1 root root 331367 Jun  4 08:43 /var/log/packages/mate-themes-3.22.17-noarch-1_msb
-rw-r--r-- 1 root root  43279 Jun 25 12:13 /var/log/packages/mate-utils-1.20.1-x86_64-1_msb
bash-4.4$``

**Comment by willysr** [comment]
Try to rebuild libmateweather first and if it doesn't work, then try mate-panel

**Comment by willysr** [comment]
also, it's advisable to upgrade everything (including kernel and glibc) because they are basic toolchain and every apps will be linked to the glibc.

**Comment by willysr** [comment]
so far, it hasn't broken all my machines yet :)
all my laptop and desktop at home are running fully updated -current machine
